homeassistant/components/tesla_fleet/const.py

```diff
@@ -32,6 +33,8 @@
     "3": "Model 3",
     "X": "Model X",
     "Y": "Model Y",
+    "C": "Cybertruck",
+    "T": "Semi",
```

Comment:
Is there a reason for this change?
I couldn't find anything related to the new models in the API bump

Reply:
I noticed those models were missing, so broke the perfect PR rule and put them in too.

Reply:
Yeah, sorry but please remove them.

Reply:
Done, I'll do a separate PR.